doc/lsst.pipe.tasks/deblending-flags-overview.rst

```diff
@@ -0,0 +1,153 @@
+#########################
+Deblending Flags Overview
+#########################
+
+When performing any analysis on an :py:class:`lsst.afw.table.SourceCatalog` it is important to understand the different subsets of sources available to ensure that the sources being analyzed are unique and to understand the biases associated with them. The aim of this document is to help you understand the different collections of sources generated by the deblender and the flags available to select them, with a little background into where the source come from and why these flags exist.
+
+Quick Reference
+===============
+
+Below is a lookup table to show the relationship between all of the different flags and subset of a `SourceCatalog`, placed here to make it easy to find in the future after you have read and understood the remainder of this document.
+
+Datasets           isPatchInner  isTractInner  isIsolated  fromBlend  isDeblendedSource  isDeblendedModelSource  isPrimary
+---------------------------  ------------  ------------  ----------  ---------  -----------------  ----------------------  ---------
+      isolated sources            x             x            x                          x                                      x
+parents with multiple peaks       x             x
+ models of isolated sources       x             x            x                                               x
+models of deblended sources       x             x                        x              x                    x                 x
+        sky objects               x             x            x           x              x                    x
+   patch overlap regions                        x            x           x              x                    x
+   tract overlap regions          x                          x           x              x                    x
+---------------------------  ------------  ------------  ----------  ---------  -----------------  ----------------------  ---------
+
+Deblender input
+===============
+
+The input `mergeDet` catalog to the deblender contains a list of parent sources, each row consisting of information about the parent, a footprint (a boolean mask of pixels in the input image that were detected as part of the parent blend), and a peak catalog (a list of peak locations and central flux values for all detected peaks in the parent blend). From here things change depending on which deblender is used, as single visit deblending uses the single-band :py:module:`lsst.meas.deblender` while multi-band deblending uses :py:module:`lsst.meas.extensions.scarlet` and generates a slightly different set of outputs.
```

Comment:
```suggestion
The input ``mergeDet`` catalog to the deblender contains a list of parent sources, each row consisting of information about the parent, a footprint (a boolean mask of pixels in the input image that were detected as part of the parent blend), and a peak catalog (a list of peak locations and central flux values for all detected peaks in the parent blend). From here things change depending on which deblender is used, as single visit deblending uses the single-band :doc:`/modules/lsst.meas.deblender` while multi-band deblending uses :doc:`lsst.meas.extensions.scarlet` and generates a slightly different set of outputs.
```

Also 

> The input mergeDet catalog to the deblender [..]

Is mergeDet the name of an argument? to a method/function? Perhaps rephrase for clarity.

Reply:
Neither, `mergeDet` is the name of an input catalog from the Butler.